http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -369,6 +368,15 @@ object clusterQuery extends TableQuery(new ClusterTable(_)) {
     }
   }
 
+  def getActiveClusterRecordByName(project: GoogleProject,
```

Comment:
Curious why we need this instead of `getActiveClusterByNameMinimal` which seems to be equivalent.. our usual practice is to not return `*Record` objects outside of DB code

Reply:
I added this because originally we were using `getActiveClusterByNameMinimal`, which stubs out fields outside cluster table. I didn't want to keep using `getActiveClusterByNameMinimal` because this caused a bug where I was using dataproc instances from `Runtime` object, but because it was stubbed out, it wasn't doing what I expected it's doing. By using  `getActiveClusterRecordByName`, the return type is exactly what it is, and there's no chance that caller might be using a stubbed out field